setup.py

```diff
@@ -21,18 +24,25 @@
 with open('README.rst') as f:
     readme = f.read()
 
+PY3 = (sys.version_info[0] >= 3)
+
+install_requires = [
+    "pyOpenSSL >= 0.13",
+    "requests >= 2.1.0",
+    "service_identity >= 14.0.0",
+    "six"
+]
+
+if PY3:
```

Comment:
I have an if: in setup.py, and proper ones in setup.cfg, so setup.cfg overwrites it to make a good set of deps, as how Twine does it? Or is this even more screwed up than I thought?